### PR TITLE
add unittest for addressSet

### PR DESCRIPTION
**Pull request by kongliangzhong**

### CODE REVIEW

File: packages/hebao_v1/test/lib/testAddressSet.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      await addressSetWrapper.add(key1, address1, false);
+      const isInSet1 = await addressSetWrapper.isInSet(key1, address1);
+      assert.equal(isInSet1, true, "address not in set after insertion");
+      await expectThrow(addressSetWrapper.add(key1, address1, false), "ALREADY_IN_SET");
+    });
+
+  });
+
+  describe("removeAddressFromSet", () => {
+    it("should be able to remove address in set", async () => {
```
Comment (adamchen0510) [inline packages/hebao_v1/test/lib/testAddressSet.ts:62]: add test  NOT_IN_SET?

File: packages/hebao_v1/test/lib/testOwnerManagable.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    const ownerManagable = await OwnerManagable.new();
+    const owner = accounts[0];
+    const address1 = web3.eth.accounts.create().address;
+    await ownerManagable.addManager(address1, {from: owner});
+    const isManager = await ownerManagable.isManager(address1);
+    assert.equal(true, isManager, "add manager failed");
+
+    await ownerManagable.removeManager(address1, {from: owner});
+    const isManager2 = await ownerManagable.isManager(address1);
+    assert.equal(false, isManager2, "add manager failed");
```
Comment (adamchen0510) [inline packages/hebao_v1/test/lib/testOwnerManagable.ts:18]: remove manager failed

File: packages/hebao_v1/test/lib/testAddressSet.ts (hunk 1 above)
Comment (kongliangzhong) [reply packages/hebao_v1/test/lib/testAddressSet.ts:62]: Added

File: packages/hebao_v1/test/lib/testOwnerManagable.ts (hunk 2 above)
Comment (kongliangzhong) [reply packages/hebao_v1/test/lib/testOwnerManagable.ts]: fixed

**Pull request merged**